airbyte-integrations/connectors/source-tmdb/source_tmdb/tmdb.yaml

```diff
@@ -0,0 +1,189 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://api.themoviedb.org/3/"
```

Comment:
Can you implement only the latets version (v4): https://developers.themoviedb.org/4/getting-started/authorization
and remove all the api_key requests in the streams.

Reply:
But the v4 doesn't support movies. Also it is in development stage, thus will have bugs right?

Reply:
Agree, so focus in using the auth for v3.